lib/model/QubitNote.php

```diff
@@ -22,6 +22,36 @@ class QubitNote extends BaseNote
     // Flag for updating search index on save or delete
     public $indexOnSave = true;
 
+    public function __get($name)
+    {
+        $args = func_get_args();
+
+        if ('type' === $name && true === $options['sourceCulture']) {
+            /*
```

Comment:
This block should get the same output as the sql query below but it seems to be getting results for a record with a different typeId?

Reply:
I would drop this ORM query in favour of the QubitPdo query you have below - the QubitPdo query will be faster, and will deliver just the result you need instead of a full instantiated object.